Core/Mechanics/Skill.cs

```diff
@@ -24,7 +25,20 @@ public abstract class Skill
 	/// <returns>The newly generated skill.</returns>
 	public static Skill GetAndPrepareSkill(Type type)
 	{
+		if (type is null)
+		{
+			ModContent.GetInstance<PathOfTerraria>().Logger.Error($"Loading Skill not found. Defaulting to Berserk.");
+			return new Berserk();
+		}
+
 		var skill = Activator.CreateInstance(type) as Skill;
+
+		if (skill is null)
+		{
+			ModContent.GetInstance<PathOfTerraria>().Logger.Error($"Skill of type {type} not found. Defaulting to Berserk.");
+			return new Berserk();
```

Comment:
Defaulting to berserk seems a bit weird but fine for now

Reply:
It's just the most "default" Skill. In the future we can probably make a placeholder skill that does nothing but hold info related to unloaded skill if necessary.